Add Rupin Mittal to contributors.json

RupinMittal opened the pull request:
#### 007636dcbb9bbbefdbad090d5fecee9a0e328828
<pre>
Add Rupin Mittal to contributors.json
<a href="https://bugs.webkit.org/show_bug.cgi?id=242255">https://bugs.webkit.org/show_bug.cgi?id=242255</a>

Reviewed by Jonathan Bedard.

Add Rupin Mittal to contributors.json.

* metadata/contributors.json:

Canonical link: <a href="https://commits.webkit.org/252068@main">https://commits.webkit.org/252068@main</a>
</pre>

szewai reviewed (changes requested):
Change looks good; I think we want the entries to be sorted by "name" alphabetically, so let's move the new entry to the middle of the file

RupinMittal committed:
Add Rupin Mittal to contributors.json
https://bugs.webkit.org/show_bug.cgi?id=242255

Reviewed by Jonathan Bedard.

Add Rupin Mittal to contributors.json.

* metadata/contributors.json:

Canonical link: https://commits.webkit.org/252068@main
